src/Tests/Backend/Backend.test.ts

```diff
@@ -35,7 +35,14 @@ class BackendMockup implements Backend {
   executor(): Executor|undefined {
     return new ExecutorBase();
   }
-};
+}
+
+const executorName = 'Mockup';
+class ExecutorMockup extends ExecutorBase {
+  name(): string {
+    return executorName;
+  }
+}
 
 suite('Backend', function() {
   suite('backendRegistrationApi', function() {
```

Comment:
Check after backend registration? (Did I understood the code correctly?)

```ts
assert.strictEqual(globalExecutorArray.length, 1);
```

Reply:
Oh I will add this. Thanks!